docs/content/administration/_index.md

```diff
@@ -21,6 +21,14 @@ The **Collectives** app can be installed from the [Nextcloud App Store](https://
 In your Nextcloud instance, simply navigate to **»Apps«**, find the
 **»Collectives«** app and enable it.
 
+## Collectives and server-side encryption
+
+With server-side encryption enabled, the files in a Collective will be stored
+encrypted on the filesystem as well.
```

Comment:
Maybe even reference https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/encryption_configuration.html#files-not-encrypted here as previews would also be relevant for collectives